package.json

```diff
@@ -44,7 +44,7 @@
     "nodeify": "^1.0.0"
   },
   "devDependencies": {
-    "aegir": "^9.4.0",
+    "aegir": "^10.0.0",
```

Comment:
Can you remove the `--webworker` flags in the scripts please, as they are not needed anymore